examples/pyramid.cpp

```diff
@@ -25,6 +25,8 @@
 /// \brief this example shows how pyramid node works.
 ///
 
+#ifdef NOT_COMPATIBLE_WITH_COMPUTECPP_050
```

Comment:
So one thing I'd say here is that we don't really care about old releases - we won't be supporting them, and other SYCL projects (the SDK, SYCL-DNN) are tagging commits to indicate compatibility.

Reply:
Yeah, that makes sense. I am trying to rework it since the separable convolution OP used for the pyramid just doesn't return the correct data like it used to do long ago but that will be on a separate PR. I think this wouldn't have happened if the repo was tagged accordingly with every new ComputeCpp release.

Reply:
For sure, this repo could do with some love!